Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-kubernetes-metrics (1.1.0)
+    fluent-plugin-kubernetes-metrics (1.1.1)
```

Comment:
Do we need to update the Gemfile.lock if there are no updates in dependencies?

Reply:
Gemfile.lock has been removed from this PR